pkg/metrics/backup.go

```diff
@@ -195,6 +198,31 @@ func (rm RestoreM) DecreaseRemainingBytes(clusterID uuid.UUID, snapshotTag strin
 	rm.remainingBytes.With(l).Sub(float64(restoredBytes))
 }
 
+// RestoreLabels is a set of labels for restore metrics.
+type RestoreLabels struct {
```

Comment:
Nice.
The same set of labels is used by remaining_bytes metric.
Please update method signature to accept `RestoreLabels` instead of `clusterID uuid.UUID, snapshotTag string, location backupspec.Location,
	dc, node, keyspace, table string,`